app/javascript/components/gtl/DataTable.jsx

```diff
@@ -50,7 +51,12 @@ export const DataTable = ({
              name={'selectAll'}
              key={'selectAll'}
              checked={!!checkedItems['selectAll']}
-             onChange={event => localOnSelectAll(rows, event)}
+             onChange={localOnSelectAll}
+             onKeyPress={localOnSelectAll}
+             role="checkbox"
+             aria-checked="false"
+             tabIndex="0"
+             aria-labelledby={'selectAll'}
```

Comment:
This is a redundant syntax, you can just use:
```suggestion
             aria-labelledby="selectAll"
```